README.md

```diff
@@ -30,11 +30,16 @@ KERNEL_CLASS="Symfony\Cmf\Bundle\ResourceRestBundle\Tests\Fixtures\App\Kernel" .
 
 
 ## Requirements
-
+Version <= 1.1:
```

Comment:
lets just remove here everything except the line 
```
 See the `require` section of [composer.json](composer.json)
```

in line with other cmf bundles: https://github.com/symfony-cmf/routing-bundle/blob/master/README.md